testing/tests/DevExpress.ui.widgets/sortable.tests.js

```diff
@@ -3599,3 +3599,77 @@ QUnit.module('autoscroll', getModuleConfigForTestsWithScroll('#itemsWithScroll',
     });
 
 });
+
+// T971119
+QUnit.module('Check bounds for container with padding', crossComponentModuleConfig, () => {
+    QUnit.test('before left', function(assert) {
```

Comment:
You use separate module, so you can move next 10 lines (changing padding and sortable, onReorderSpy and $sourceElement creation) to the beforeEach.